src/TextField.tsx

```diff
@@ -170,6 +174,10 @@ const Text = (props: TextFieldProps) => {
         value={props.value ?? ''}
         {...filterDOMProps(props)}
       />
+      <HelperText>
+        <HelperTextItem>{props.helperText}</HelperTextItem>
+      </HelperText>
```

Comment:
Typically any new feature we add should not affect dom in any way so integration tests would not break. 
In this case we can get away with introducing another div component but we should not add empty helper text.
```suggestion
{props.helperText &&
      <HelperText>
        <HelperTextItem>{props.helperText}</HelperTextItem>
      </HelperText>}
```